### PR TITLE
feat: release v3.2.0

### PR DESCRIPTION
**Pull request by stefanoghinelli**
Changes included in this PR:
- update docs (readme, comp matrix, release notes)
- update tests
- refactor CI to use Kind

Resolves: [504](https://github.com/sighupio/product-management/issues/504)

### CODE REVIEW

**Review by ralgozino — CHANGES_REQUESTED** [review]
overwall great job!

is this PR just for the CI and you will open another one bumping harbor version or are we not updating harbor for some reason?

File: .drone.yml [hunk 1]
```diff
@@ -504,206 +347,124 @@ trigger:
...
-        from_secret: dynamic_dns_service
+      KUBECONFIG: kubeconfig-harbor-v1.30.4
+      TEST_DOMAIN: "127.0.0.1.nip.io"
+    depends_on:
+      - create-kind-cluster
     commands:
-      - export KUBECONFIG=/shared/kube/kubeconfig-128
-      - export INSTANCE_IP=$(cat /shared/machine/ip)
-      - export EXTERNAL_DNS="$INSTANCE_IP.$DYNAMIC_DNS_SERVICE"
+      - curl -LOs https://github.com/sighupio/furyctl/releases/download/v0.11.1/furyctl-linux-amd64 && chmod +x furyctl-linux-amd64 && mv furyctl-linux-amd64 /usr/local/bin/furyctl
```
[the remark this answers is not in the document]
Comment (stefanoghinelli) [reply .drone.yml:377]: Nope, the e2e-testing image in 1.28 and 1.29 pipelines has furyctl `0.11.0` pre-installed. To maintain consistency and use the same `furyctl vendor -H `command used in `setup.sh` tests, we need to keep using version `0.11.0` in the newer as well.

[the remark this answers is not in the document]
Comment (stefanoghinelli) [reply .drone.yml]: Done.

File: .drone.yml [hunk 2]
```diff
@@ -86,203 +90,124 @@ trigger:
...
-      vsphere_user:
-        from_secret: vsphere_user
-    when:
-      ref:
-        include:
-          - refs/tags/**
+      - name: dockersock
+        path: /var/run/docker.sock
+    environment:
+      KIND_CONFIG: ./katalog/tests/harbor/config/kind-config
```
[the remark this answers is not in the document]
Comment (stefanoghinelli) [reply .drone.yml]: Done.

**Comment by stefanoghinelli** [comment]
Talking with Sam, I understood that bumping harbor version is out of scope for this release and not currently required. Right @nutellinoit?

**Review by ralgozino — CHANGES_REQUESTED** [review]
> Talking with Sam, I understood that bumping harbor version is out of scope for this release and not currently required. Right @nutellinoit?

I mean, that's totally fine, but if that is the case then we don't need to do a new release, because there is no point in updating the module from a user perspective because there are absolutely no changes.

Checking if upstream is compatible, updating the tests, docs, etc. (i.e. what you did in this PR) is enough to have the module compatible with the newer versions IMO.

So, I'd say to remove the release notes for 3.2.0 and this PR is 🟢 .

Alternatively, if we want/need to do a release, let's be more specific in the release notes that the release does not practically change anything and that can be ignored.

**Comment by stefanoghinelli** [comment]
As discussed, I updated the release notes to clarify that this release is purely for Kubernetes compatibility testing and doesn't include changes.

**Review by ralgozino — APPROVED** [review]
perfect!

**Pull request merged**